test/simple.test.ts

```diff
@@ -43,6 +46,47 @@ integrationTestSkipOnWindows("forwarding signals to the child", async () => {
   expect(readAppmap()).toMatchSnapshot();
 });
 
+integrationTestSkipOnWindows(
+  "forwards SIGINT (ctrl-c) properly to the grandchild having active setInterval",
+  async () => {
+    // This test tests a fix (see: forwardSignals.ts) for a bug which occured
+    // when there is a script with active setInterval call and appmap-node
+    // runs this script indirectly with a node process: appmap-node node interval.js.
+    // We run the commands similarly as reported in:
+    // https://github.com/getappmap/appmap-node/issues/118
+    const child = spawn("npx", [getAppMapBinPath(), "node", "interval.js"], {
```

Comment:
Why `npx` instead of `node`?

Reply:
I was able to consistently create same buggy behavior only with `npx` in `Jest`. Considering that original command reported in the issue was using `npx`, I believe it represents the core problem.

Substituting `node` for `npx` in the test does not fail but I've also manually tested it with `node`, as you suggested in the issue: It fails without the fix and succeeds with the fix.

On the other hand, since the issue revolves around properly handling signals (Ctrl-C in this case) across multiple processes, I'm uncertain whether it's possible to simulate exact same circumstances of receiving Ctrl-C in [`npx` -> `node appmap-node` -> `node intervalScript.js`] vs. [`Jest Runner` -> `shell` -> `npx` -> `node appmap-node` -> `node intervalScript.js`]. This configuration was the closest I could set up in a `Jest` test.